api/envoy/service/auth/v3/external_auth.proto

```diff
@@ -96,6 +96,11 @@ message OkHttpResponse {
   // setting this field overrides :ref:`CheckResponse.dynamic_metadata
   // <envoy_v3_api_field_service.auth.v3.CheckResponse.dynamic_metadata>`.
   google.protobuf.Struct dynamic_metadata = 3 [deprecated = true];
+
+  // This field allows the authorization service to send HTTP response headers
+  // to the downstream client on success. Note that the `append` field in `HeaderValueOption`
```

Comment:
A link back to `HeaderValueOption` would be appreciated.

Reply:
Will do

Reply:
Done. I updated the other places in this proto that referred to HeaderValueOption as well.

Reply:
Thanks!